content/events/2020-amsterdam/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Amsterdam
+date: '2020-09-24'
+end: '2020-09-25'
+location:
+  country: The Netherlands
+  city: Amsterdam
+  url: https://www.amsterdam.nl/
+  venue: Westergasterras
+map: amsterdam-map.png
+sched_slug: kubernetescommunitydayams
+social:
+  facebook: https://facebook.com/cloudnativeams
+  twitter: 'https://twitter.com/cloudnativeams'
+sponsors:
+- level: Initial committers
+  orgs:
+  - name: Microsoft
+    logo: sponsors/microsoft.svg
+    url: https://microsoft.com
+  - name: Container Solutions
+    logo: sponsors/container-solutions.svg
+    url: https://www.container-solutions.com/
+  - name: Fullstaq
+    logo: sponsors/fullstaq.png
+    url: https://fullstaq.com/
```

Comment:
Please don't use a PNG. Logo is here: https://landscape.cncf.io/logos/fullstaq-member.svg

Please try running the SVGs through autocrop.cncf.io so that they line up.

Reply:
addressed in 457cb0e